### DISCUSSION TITLE
登录异常

### DISCUSSION THREAD

**Discussion by nizpxox** [zuihou/zuihou-admin-cloud#311] [01.常见问题]
![image](https://github.com/dromara/lamp-cloud/assets/141626156/b2d36cd0-da5c-49e6-a577-56f868f5dc73)
部署到服务器上了
![image](https://github.com/dromara/lamp-cloud/assets/141626156/7e7b7289-fc75-4f24-ba71-6024e992aed2)
![image](https://github.com/dromara/lamp-cloud/assets/141626156/f4c27d0e-ccf1-48cb-8ba7-ca548bef8b31)

**Comment by zuihou** [comment]
看下 登录人的 base_org 表的 TreePath 字段是什么
